numpy/lib/_function_base_impl.py

```diff
@@ -5418,43 +5418,49 @@ def insert(arr, obj, values, axis=None):
     -----
     Note that for higher dimensional inserts ``obj=0`` behaves very different
     from ``obj=[0]`` just like ``arr[:,0,:] = values`` is different from
-    ``arr[:,[0],:] = values``.
+    ``arr[:,[0],:] = values``. This is because of the difference between basic
+    and advanced :ref:`indexing <basics.indexing>`.
 
     Examples
     --------
-    >>> a = np.array([[1, 1], [2, 2], [3, 3]])
+    >>> a = np.arange(6).reshape(3, 2)
```

Comment:
Swapping to a different example which has no repeated numbers hopefully helps make it obvious which things are changing. The rest of the changes below are to make sure newly inserted objects are different.